### DISCUSSION TITLE
How to listen for Textbox width and height changes

### DISCUSSION THREAD

**Discussion by Ezr1al** [fabricjs/fabric.js#7554] [Q&A]

**Comment by Ezr1al** [comment]
I need to know when the width or height of the Textbox changes,How to listen for Textbox width and height changes?

**Reply by asturur** [reply]
i think we have events? `resizing` is the event you are looking for.
There is no height change, that would be a `changed` event on the text content.

textbox.on('resizing', yourHandler)
